src/OrbitQt/orbitlivefunctions.ui

```diff
@@ -45,88 +45,109 @@
        </sizepolicy>
       </property>
      </widget>
-     <widget class="QWidget" name="verticalLayoutWidget">
-      <layout class="QVBoxLayout" name="verticalLayout_3">
-       <item>
-        <widget class="QLabel" name="instructionsLabel">
-         <property name="styleSheet">
-          <string notr="true">margin-top: 5px;</string>
-         </property>
-         <property name="text">
-          <string>Select functions above, right-click, and select 'Add iterator(s)' to add iterators for instrumented functions.</string>
-         </property>
-         <property name="margin">
-          <number>0</number>
-         </property>
-        </widget>
-       </item>
-       <item>
-        <widget class="QScrollArea" name="iteratorScrollArea">
-         <property name="autoFillBackground">
-          <bool>true</bool>
-         </property>
-         <property name="styleSheet">
-          <string notr="true">QScrollArea {
-	border: 0;
-}</string>
-         </property>
-         <property name="sizeAdjustPolicy">
-          <enum>QAbstractScrollArea::AdjustToContents</enum>
-         </property>
-         <property name="widgetResizable">
-          <bool>true</bool>
-         </property>
-         <widget class="QWidget" name="iteratorFrame">
-          <property name="geometry">
-           <rect>
-            <x>0</x>
-            <y>0</y>
-            <width>688</width>
-            <height>187</height>
-           </rect>
+     <widget class="QTabWidget" name="tabWidget">
+      <property name="currentIndex">
+       <number>0</number>
+      </property>
+      <widget class="QWidget" name="histogram_tab">
```

Comment:
I have some UI thought here.
In general I'm not a fan of tabs inside tabs as it gets a bit convoluted. I was wondering if we really need the tabs here.
The vertical space taken by iterators is quite limited when none is set (a lot of space is currently wasted actually but we can fix that). Also, the vertical space for the histogram could be completely freed up when no histogram is set (we can have a string saying "Select a function to show the histogram" or something like that for discoverability, similarly to the string for iterators). So could they just be one on top of the other with a better use of space?

Reply:
I don't have a strong opinion. @reichlfl WDYT? If I remember correctly, you've suggested tabs under tabs.

Reply:
But certainly I like your discoverability idea.

Reply:
But yeah, it was just a thought, I don't know it would be better, I can't really imagine it that well in my head.

Reply:
What percentage of our users actually care for the iterators? Is it high enough to allow the iterators to occupy at least an inch of the space by default? I've got am impression, most of the users will be happy to see the histograms and will not miss the iterators. Or is there any value in seeing histograms and iterators at the same time?